### ISSUE TITLE
Cannot access lamby-cookiecutter image

### ISSUE THREAD

**Issue by jrhoads** [rails-lambda/lamby-cookiecutter#38]
First: Thank you for your work on this very cool project.

I'm probably messing this up myself but when I enter the very first command in the [quickstart](https://lamby.cloud/docs/quick-start)

```
docker run \
  --rm \
  --interactive \
  --volume "${PWD}:/var/task" \
  ghcr.io/rails-lambda/lamby-cookiecutter \
  "gh:rails-lambda/lamby-cookiecutter"
```

Docker is unable to find image 'ghcr.io/rails-lambda/lamby-cookiecutter:latest' locally (reasonable).  But when it goes out to ghcr.io to get the image I get the following error

`docker: Error response from daemon: Head "https://ghcr.io/v2/rails-lambda/lamby-cookiecutter/manifests/latest": unauthorized.`

Did I miss something or did a switch get flipped and now the ghcr repo needs authentication?

**Comment by metaskills** [comment]
Hey @jrhoads, try it now. Should be fixed.

Short story, you caught us in the middle if a GitHub org migration and I forgot to deploy an updated container package to the cookie cutter repo. Should be here now. https://github.com/rails-lambda/lamby-cookiecutter/pkgs/container/lamby-cookiecutter

**Comment by jrhoads** [comment]
Hey @metaskills , thanks for taking a look at this.

I tried the command again and received the same result.  The link gives me a 404 as well.

I did have some success skipping the docker part and installing cookiecutter directly through pipx `pipx install cookiecutter`  then I was able to just use `cookiecutter "gh:rails-lambda/lamby-cookiecutter"`

I'm happy to try this again if you have time to work on it for the sake of the docs, but for my purposes I'm good with using cookiecutter directly.
